### PR TITLE
WIP: Autodetection of latest version and updated after-installation message

### LINKED ISSUE #24: Improve get.sh script with similar features like istio quickstart has
Source: https://raw.githubusercontent.com/istio/istio/master/release/downloadIstioCandidate.sh

Examples:

**Automatically determine version based on latest gh release**
```
# Determine the latest Istio version by version number ignoring alpha, beta, and rc versions.
if [ "x${ISTIO_VERSION}" = "x" ] ; then
  ISTIO_VERSION="$(curl -sL https://github.com/istio/istio/releases | \
                  grep -o 'releases/[0-9]*.[0-9]*.[0-9]*/' | sort --version-sort | \
                  tail -1 | awk -F'/' '{ print $2}')"
  ISTIO_VERSION="${ISTIO_VERSION##*/}"
fi

if [ "x${ISTIO_VERSION}" = "x" ] ; then
  printf "Unable to get latest Istio version. Set ISTIO_VERSION env var and re-run. For example: export ISTIO_VERSION=1.0.4"
  exit;
fi
```

**Print some helpful information on how to continue**

```
printf ""
printf "\nIstio %s Download Complete!\n" "$ISTIO_VERSION"
printf "\n"
printf "Istio has been successfully downloaded into the %s folder on your system.\n" "$NAME"
printf "\n"
BINDIR="$(cd "$NAME/bin" && pwd)"
printf "Next Steps:\n"
printf "See https://istio.io/latest/docs/setup/install/ to add Istio to your Kubernetes cluster.\n"
printf "\n"
printf "To configure the istioctl client tool for your workstation,\n"
printf "add the %s directory to your environment path variable with:\n" "$BINDIR"
printf "\t export PATH=\"\$PATH:%s\"\n" "$BINDIR"
printf "\n"
printf "Begin the Istio pre-installation check by running:\n"
printf "\t istioctl x precheck \n"
printf "\n"
printf "Need more information? Visit https://istio.io/latest/docs/setup/install/ \n"
```

### PR DESCRIPTION
**Pull request by adriangonciarz**
https://github.com/keptn/get.keptn.sh/issues/24

### CODE REVIEW

File: src/get.sh [hunk 1]
```diff
@@ -1,7 +1,53 @@
 #!/bin/sh
 
-# latest GA release
-KEPTN_VERSION=${KEPTN_VERSION:-"0.7.3"}
+# Define handy functions
+get_latest_version(){
+   curl --silent "https://api.github.com/repos/keptn/keptn/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'
```
Comment (christian-kreuzberger-dtx) [inline src/get.sh]: This command is not working on MacOS, see https://travis-ci.com/github/keptn/get.keptn.sh/jobs/461956400

I thinkt he problematic part is the `sed -E` command. Take a look at the script I copied from istio here https://github.com/keptn/get.keptn.sh/issues/24, they are not using sed, but a combination of grep and tail.

Comment (jetzlstorfer) [reply src/get.sh]: I verified on Mac OS 10.14.6 and it is working. 
```
juergen.etzlstorfer$ curl --silent "https://api.github.com/repos/keptn/keptn/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'
0.7.3
```

Which OS version is used in travis?

Comment (christian-kreuzberger-dtx) [reply src/get.sh]: Thanks Jüergen. I think travis-ci has a weird version of awk available. Though I'm not sure if people might have this version, or if everyone has the correct version ...

Anyway, Travis-CI osx is running 
```
Operating System Details
ProductName:	Mac OS X
ProductVersion:	10.13.6
BuildVersion:	17G65
```

**Comment by christian-kreuzberger-dtx** [comment]
@agrimmer @bacherfl when you have time, can you take a look why the awk and/or sed commands proposed in this PR are failing on MacOS? I believe it might just be an issue with travis-ci, but I want to rule out that this is an issue with MacOS in general.

If we merge this, will people with old MacOS versions not be able to run the quickstart?

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**